### PR TITLE
[unixODBC] Add new port

### PR DESCRIPTION
**Pull request by kreuzerkrieg**
- What does your PR fix?
Add unixODBC port
- Which triplets are supported/not supported?
No Win/UWP
- Have you updated the CI baseline?
Yes
- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

### CODE REVIEW

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks for the PR!

I will help rerun after the CI baseline issue fix.

**Comment by kreuzerkrieg** [comment]
Anything prevents this PR from being merged?

File: ports/unixodbc/CONTROL [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Source: unixodbc
+Version: 2.3.7
+Homepage: https://github.com/lurcher/unixODBC
+Description: unixODBC is an Open Source ODBC sub-system and an ODBC SDK for Linux, Mac OSX, and UNIX
```
Comment (strega-nil) [inline ports/unixodbc/CONTROL]: Could you add a `Supports` clause here?

**Comment by strega-nil** [comment]
Alright, we're currently waiting on the qt tools to be fixed in the CI, then we'll merge :)

**Comment by PhoebeHui** [comment]
After rerun, the CI test pass.

**Comment by strega-nil** [comment]
Awesome, thanks @kreuzerkrieg for all this work you're doing!

**Pull request merged**